index.html

```diff
@@ -1,304 +1,243 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="styles.css" />
-    <link
-      href="https://fonts.googleapis.com/icon?family=Material+Icons"
-      rel="stylesheet"
-    />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">      
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
   </head>
   <body>
-    <div class="main-container">
-      <nav class="nav-bars">
-        <div class="logo">
-          <a href="index.html"><h2>Duncan</h2></a>
+      <div id="header">
+      <header id="navigation-bar">
+        <div id="logo">
+          <a href="#header"><img src="img/MyLogo.png" alt="Logo"></a>
         </div>
-        <nav class="menu">
-          <nav><img src="photo/navs/Union.png" alt="" /></nav>
+        <nav id="nav">
+          <div id="btn-menu"><img src="img/Normal-Button/Tertiary/Icons/Icon-Menu.png" alt=""></div>
+          <ul id="menu-lst">
+            <li><a href="#headline" class="menu-lst-txt">Portfolio</a></li>
+            <li><a class="menu-lst-txt" href="#About-my-self">About</a></li>
+            <li><a class="menu-lst-txt" href="#Contact-Section">Contact</a></li>
+          </ul>
         </nav>
-      </nav>
-
-      <section class="About-me">
-        <div class="greeting">
-          <h1>I'm Duncan Glad to see you!</h1>
-        </div>
-
-        <div class="biography">
-          <p>
-            I’m a software developer! I can help you design and build a product
-            , feature or website Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don’t
-            hestiate to contact me.
+      </header>
+      <section id="headline">
+        <div class="headline-txt">
+          <h1 class="txt-1 order-0 pd-0 mrg-0 mbl-to-hide h1-animation">I’m Duncan Glad to see you!</h1>
+          <h1 class="txt-1-dsktp order-0 pd-0 mrg-0 dsktp-to-hide dsktp-to-show h1-animation">I’m Duncan <br>
+            Glad to see you!</h1>
+          <p class="txt-2-dsktp order-1 pd-0 mrg-0 txt-j">I’m a software developer! I can help you build
+             a product , feature or website Look through
+             some of my work and experience! If you
+             like what you see and have a project you need
+             coded, don’t hestiate to contact me.
```

Comment:
- [ ] Good job on your UI design 👍🏽 . I think it will be better if you move your headline section upward a bit for the desktop version by either removing the margin-top or reducing it so that a user can see first-hand the content of the headline section without a need to scroll down which is a good user experience 😄 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/187036558-d14347d5-604f-4ffc-bf6d-1bef0de51088.png)  | ![image](https://user-images.githubusercontent.com/63473584/187036600-ac5e16ba-9526-4d9f-89cb-2da334e431ff.png)  |

Reply:
Hey @Duncannjau, as requested by the previous reviewer please fix this issue of spacing above the main heading. It is super important to follow the Figma template of your choice for this module. Kindly implement the change and request a re-review. You might want to change the margins for the `headline-txt` class from 20% to 10% or so. That should fix the issue.
![image](https://user-images.githubusercontent.com/33268927/187042179-01bc78e4-b416-4205-8e12-099f05ef5d39.png)